apps/allocations/app/components/App/App.js

```diff
@@ -42,14 +71,21 @@ const App = () => {
     closePanel()
   }
 
+  const onSubmitDeactivate = (id) => {
+    console.log(`deactivating budget # ${id}...`)
+    //api.deactivateBudget(id)
```

Comment:
Thanks for adding this here! I will take care of removing it during the integration process, this kind of practice really helps on this case.